photon-core/src/main/java/org/photonvision/vision/pipe/impl/NeuralNetworkPipeResult.java

```diff
@@ -20,10 +20,10 @@
 import org.opencv.core.Rect2d;
 
 public class NeuralNetworkPipeResult {
-    public NeuralNetworkPipeResult(Rect2d box2, Integer classIdx, Float confidence) {
+    public NeuralNetworkPipeResult(Rect2d box2, int classIdx, double confidence2) {
         box = box2;
         this.classIdx = classIdx;
-        this.confidence = confidence;
+        this.confidence = confidence2;
```

Comment:
whats the point of changing the name here, not that it matters but

Reply:
Vscode moment. No reason, I'll fix it.